kogito-codegen/src/main/java/org/kie/kogito/codegen/process/persistence/PersistenceGenerator.java

```diff
@@ -66,8 +66,8 @@
 
     private static final String FILESYSTEM_PERSISTENCE_TYPE = "filesystem";
     private static final String INFINISPAN_PERSISTENCE_TYPE = "infinispan";
```

Comment:
just to make it consistent with the other tyes
```suggestion
    public static final String FILESYSTEM_PERSISTENCE_TYPE = "filesystem";
    public static final String INFINISPAN_PERSISTENCE_TYPE = "infinispan";
```